meetings/community.yaml

```
@@ -0,0 +1,13 @@
+project: Community working group
+meeting_id: community
+project_url: https://github.com/ansible/community/wiki/Community
+agenda_url: https://github.com/ansible/community/issues?q=is:open+label:meeting_agenda+label:contributor_experience
+chair: John Barker (gundalow)
+description: |
+
+    Discuss everything related to Ansible Community and Contributor experience.
+schedule:
+- time: '1700'
```

shouldn't it be:
```suggestion
- time: '1800'
```
? Now some part of the docs says 17:00 UTC, some other 18:00 UTC. I think we had it on 18:00 UTC last week (one hour earlier than the public project meeting on Tuesday).

---

Yup, good spot. Forgot I'm not in UTC at the moment